domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/simulering/Simulering.kt

```diff
@@ -89,31 +83,22 @@ enum class SimuleringFeilet {
 
 enum class KlasseType {
     YTEL,
-    SKAT
+    SKAT,
+    FEIL,
+    @Deprecated("Filtreres ut av klient") // TODO flytt dette lenger ut
+    MOTP,
```

Comment:
Har vi lagt til en ny type som er deprecated from the get go? Har vi denne i basen?